psydac/api/fem.py

```diff
@@ -123,10 +124,13 @@ def construct_trial_space_arguments(basis_values):
     return trial_basis, trial_degrees, pads
 
 #==============================================================================
-def construct_quad_grids_arguments(grid):
+def construct_quad_grids_arguments(grid, weights=True):
     points         = grid.points
-    weights        = grid.weights
-    quads          = flatten(list(zip(points, weights)))
+    if weights:
+        weights        = grid.weights
+        quads          = flatten(list(zip(points, weights)))
+    else:
+        quads = flatten(list(zip(points)))
 
     quads_degree   = flatten(grid.quad_order)
```

Comment:
no, I'm using `quad_order` now, if you want to change `quad_order` everywhere, then it's beyond the scope of this PR, because we are using it everywhere, and it was not introduced by this PR

Reply:
Yes, you are probably right. Please create a new issue!

Reply:
done  #151